features/product/viewing_products/viewing_product_sorted_by_date.feature

```diff
@@ -0,0 +1,27 @@
+@viewing_products
+Feature: Sorting listed products
+    In order to change the order by which products are displayed
+    As an Administrator
+    I want to sort products
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Berserk Pug" with code "B_PUG"
+        And this product was created at "05-10-2016"
+        And this product belongs to "Fluffy Pets"
+        And the store also has a product "Pug of Love" with code "L_PUG"
+        And this product was created at "06-10-2016"
+        And this product belongs to "Fluffy Pets"
+        And the store also has a product "Xtreme Pug" with code "X_PUG"
+        And this product was created at "07-09-2016"
+        And this product belongs to "Fluffy Pets"
+        And I am a logged in customer
+
+
+    @todo
+    Scenario: Sorting products by their dates of creations
+        When I browse products from taxon "Fluffy Pets"
+        And I start sorting products by creation date
```

Comment:
``And I sort products by theirs creation dates``? ``start`` seems like it's a long-time process 😄